docker-compose-host.yml

```diff
@@ -4,21 +4,21 @@ volumes:
   cloudbeaver: {}
   cloudbeaver_certs: {}
   postgres_data: {}
-  nginx_ssl_data: {}
-  nginx_conf_data: {}
+  web_proxy_ssl_data: {}
```

Comment:
don't rename volume because of backward compatibility (add comment here to not rename)